.github/composite_actions/log_cw_metric_wrapper/action.yaml

```diff
@@ -85,8 +85,8 @@ runs:
 
         metric-name: "github_metric_1.0"
         test-type: ${{ inputs.test-type }}
-        package-name: ${{ inputs.package-name }}
-        matrix: ${{ toJson(matrix) }}
+        working-directory: ${{ inputs.working-directory }}
+        matrix: ${{ toJson(matrix) != 'null' && toJson(matrix) || '' }}
```

Comment:
I'm not following this. Is there a way to write this more clearly in code? If not, can you add a comment.

Reply:
Good point, this code can be improved.  Updated to:

```
# Sending a null value will cause the action to fail so we must send an empty string here.
        matrix: ${{ matrix == '' && '{}' || toJSON(matrix) }}
```
        
If the matrix value is empty we send a blank {} else we convert to json and send back.